### PR TITLE
End of the update

### PR DESCRIPTION
**Pull request by Javitocatral**

### CODE REVIEW

File: src/lib/components/Header.svelte [hunk 1]
```diff
@@ -29,7 +29,7 @@
   {#if showsNavigation}
     <nav class="row {$theme && 'visible'}" transition:slide>
       <LanguageSelector {lang} />
-      <Anchor id="contact-link" sveltekit:prefetch href="/contact">
+      <Anchor id="contact-link" data-sveltekit-preload-data href="/contact">
```
Comment (zwergius) [inline src/lib/components/Header.svelte:32]: I am pretty sure that once u have added preload to body you can remove it here

File: src/lib/components/ClientsSticker.svelte [hunk 2]
```diff
@@ -97,7 +97,7 @@
     <Anchor
       id="clients-link"
       href="/cases"
-      sveltekit:prefetch
+      data-sveltekit-preload-data
```
Comment (zwergius) [inline src/lib/components/ClientsSticker.svelte:100]: I am pretty sure that once u have added preload to body you can remove it here

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/lib/components/Header.svelte [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <script lang="ts">
-  import { LL } from '$i18n/i18n-svelte'
+  import { LL } from '../../i18n/i18n-svelte'
```
[the remark this answers is not in the document]
Comment (Javitocatral) [reply src/lib/components/Header.svelte:2]: If you don't find the module

File: src/lib/components/FlipButton.svelte [hunk 4]
```diff
@@ -11,9 +11,9 @@
   class="button"
   on:click={toggleFlip}
   on:keypress={handleKeypress}
-  role="button"
+  role="switch"
   tabindex="0"
-  aria-selected={flipped ? true : undefined}
+  aria-checked={flipped ? true : undefined}
```
[the remark this answers is not in the document]
Comment (Javitocatral) [reply src/lib/components/FlipButton.svelte:16]: rol button only aria-pressed
aria selected  only this: row, tab, opiton ,gridcell

**Comment by zwergius** [comment]
Option sounds great!

**Comment by zwergius** [comment]
You should look for a solution to this

**Review by zwergius — APPROVED** [review]
LGTM
ok usa

**Pull request merged**